framework/doc/content/source/transfers/MultiAppReporterTransfer.md

```diff
@@ -73,6 +97,37 @@ Here, we are transferring integer and string data between reporters:
 !listing reporter_transfer/main.i block=int_to_int int_from_int string_from_string string_to_string caption=Main application reporter transfers
    indent=2 header=[Transfers] footer=[]
 
+
+
+### Distribution of a vector Reporter
+
+#### Scatter vector Reporter
+
+Here we are transferring a vector reporter and a vector of vectors reporter in a
+scatter fashion. The main application holds a vector of vectors and a
+single vector, while each subapp has a vector and a Real (scalar) reporter.
+During the transfer the subapp's reporters are populated from the scattering of
+the mainapp's vector reporter and a vector of vectors reporter.
+
+!listing dist_vector/main_send.i block=Reporters caption=Main application reporters
+
+!listing dist_vector/sub.i block=Reporters caption=Sub-application reporters
+
+!listing dist_vector/main_send.i block=Transfers caption=Main application reporter transfers
+   indent=2 header=[Transfers] footer=[]
+
+
+#### Gather vector Reporter
+
+Here we are transferring a vector reporter and a vector of vectors reporter in a gather fashion. The main application holds a vector of vectors and a single vector, while each subapp has a vector and a Real (scalar) reporter. This test shows the gather operation by aggregating the same values from each subapp into a set of reporters in the main application.
```

Comment:
why a set of report?
isnt it a vector report?

Reply:
```suggestion
Here we are transferring a vector reporter (and as a second example, a vector of vectors reporter) in a gather fashion. The main application holds a vector of vectors and a single vector, while each subapp has a vector and a Real (scalar) reporter. This test shows the gather operation by aggregating the same values from each subapp into a set of reporters in the main application.
```

Reply:
I used the word set loosely. Fix it to be more precise.